go.mod

```diff
@@ -6,6 +6,7 @@ replace (
 	github.com/docker/cli => github.com/docker/cli v23.0.6+incompatible
 	github.com/docker/docker => github.com/docker/docker v23.0.6+incompatible
 	github.com/kubernetes/kompose => github.com/meshery/kompose v1.26.2-0.20230425025309-3bb778d54007
+	github.com/layer5io/meshkit v0.7.18 => github.com/Jougan-0/meshkit v0.7.18
```

Comment:
If i remove the meshkit wouldn't that lead to linting error because the code is referencing some parts from the meshkit's [here](https://github.com/meshery/meshery/blob/f0087365f8a14388f4a59d54c4ba5ab01ebc1df0/server/meshmodel/helper.go#L259)

Reply:
@Jougan-0, I get that, but we're not going to be able to merge this unless it is changed. You need to 1) mark this PR a draft and do-not-merge and 2) write this caveat prominently into the description of the PR.

Reply:
Done sir